### PR TITLE
Use PHP attributes instead of annotations for Doctrine entities

### PR DESCRIPTION
**Pull request by rimas-kudelis**

### CODE REVIEW

**Comment by Ferror** [comment]
Hi @rimas-kudelis would you like to rebase the PR and I think we will be able to merge it :)

**Comment by rimas-kudelis** [comment]
@Ferror i hope you don't mind that I merged instead. If you do, I can still rebase.

**Comment by rimas-kudelis** [comment]
@Ferror done.

**Review by Ferror — APPROVED** [review]

**Review by loic425 — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
     tests:
         runs-on: ubuntu-18.04
 
-        name: "PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}"
+        name: "PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, Node ${{ matrix.node }}, MySQL ${{ matrix.mysql }}"
```
Comment (diimpp) [inline .github/workflows/build.yml:19]: Please rebase code, it's already done in current master.

**Review by diimpp — APPROVED** [review]
I'm voting for merging in 1.12, people using attributes on new projects for a while now and issues in slack `#support` about annotation/attributes are common.

File: .github/workflows/build.yml (hunk 1 above)
Comment (diimpp) [reply .github/workflows/build.yml:19]: @rimas-kudelis could you please rebase PR.

Comment (rimas-kudelis) [reply .github/workflows/build.yml:19]: @diimpp, I've done it once already. It's your turn now.

**Comment by GSadee** [comment]
I'm closing as attributes are already defined and configured 💪🏻

**Pull request closed without merging**